priv/posts/eef/20230403021727_election-2023-revote.md

```diff
@@ -0,0 +1,275 @@
+
+The EEF strives for a fair and democratic process in all voting matters and this irregularity was unintentional. In the hopes of being transparent and doing right by our members, we have therefore chosen to cancel the vote, and re-run it from scratch, even if this isn’t the most convenient way to go.
+
+To be clear, this is a re-running of the poll, no more candidate submissions shall be accepted, and only people who were entitled to vote in the original poll will be able to vote within this one.
+
+The new dates are :
+
+- Friday Apr 14th at 02:45 PM UTC to Saturday Apr 22nd at 02:45 PM UTC
+- Friday Apr 14th at 07:45 AM PDT to Saturday Apr 22nd at 07:45 AM PDT
+- Friday Apr 14th at 10:45 AM EDT to Saturday Apr 22nd at 10:45 AM EDT
+- Friday Apr 14th at 04:45 PM CEST to Saturday Apr 22nd at 04:45 PM CEST
+- Friday Apr 14th at 11:45 PM JST to Saturday Apr 22nd at 11:45 PM JST
+- Saturday Apr 15th at 12:45 AM AEST to Sunday Apr 23rd at 12:45 AM AEST
+
+The new dates have been chosen based on Section 3.4. Notice in our [bylaws](https://erlef.org/bylaws/) (There must be at least 10 days notice ahead of a special meeting). The times (now communicated in many time zones) have been chosen to be clear and will be used in all communications ahead of the poll. We have chosen deadlines that are easier to enforce without deviating from announced times.
+
+We are also going to do a more in-depth review of our process and approach to
+run elections, such that we lower the probabilities of running into more
+procedural issues in the upcoming years. We will write about them in a follow-up blog post.
+
+We look forward to seeing you in the polls and we welcome questions and comments which may be sent to
+[voting@erlef.org](mailto:voting@erlef.org)
+
+The following pages will kept up for historical record :
+
+- [The EEF Board is Having its 2023 Election](https://erlef.org/blog/eef/election-2023)
+- [The EEF Board 2023 Election is Over](https://erlef.org/blog/eef/election-2023-results)
+
+## About the election
+
+The Erlang Ecosystem Foundation is set up as a community organisation, to be run for and by the community. As such, the role of board member is assigned by a democratic process, where voting members get to run and vote for their preferred candidates.
+
+The process to be followed is described in legalese in our bylaws, but this page will explain more clearly how we plan for this to be working forwards.
+
+## Important Dates
+
+The following dates are important to the campaign:
+
+- February 25: Elections announced, people can submit their candidacy
+
+- March 10: Last day to submit candidacy. Email acceptance ends at midnight PT on the 11th
+
+- Apr 14th 02:45 PM UTC: Votes are open
+
+- Apr 22nd 02:45 PM UTC: Votes are closed
```

Comment:
useful to re-post the full list in all timezones here?

Reply:
You mean the history? I thought about that, if so, I wasn't sure it would be too confusing. I'm happy to include it if you think otherwise. Of course let me know if you mean something else.

Reply:
Done